client/src/index.jsx

```diff
@@ -1,5 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/app.jsx';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
 
-ReactDOM.render(<App />, document.getElementById('booking'));
+ReactDOM.render(
```

Comment:
@JayBo7 We don't want to render this component, I believe we want to put this as a property of the window object so that way we can use the bundle.js in our proxy server.